scripts/setup_db.py

```diff
@@ -26,6 +31,24 @@ def create_tables(hostname, dbname, user):
                 male               INTEGER NOT NULL,
                 female             INTEGER NOT NULL
             )
+            """,
+            """
+            DROP TABLE IF EXISTS service_providers
+            """,
+            """
+            CREATE TABLE service_providers (
+                id INTEGER PRIMARY KEY,
+                census_division_id INTEGER REFERENCES census_division (id),
+                name TEXT NOT NULL,
+                website TEXT,
+                report_year INTEGER,
+                report_link TEXT,
+                expenses INTEGER,
+                client_total INTEGER,
+                staff_total INTEGER,
+                address TEXT,
```

Comment:
What's our plan for satellite addresses?

Reply:
i think the addresses table should have a foreign key for `service_provider_id`. seems to be the best way to handle the one-to-many relationship.

thoughts?